os/board/Kconfig

```
@@ -284,6 +284,13 @@ config BOARDCTL_RESET
 		Architecture specific logic must provide the board_reset()
 		interface.
 
+config BOARDCTL_RESET_HEADER
+	bool
+	default n
+	---help---
+		If selected it will includes a header with board specification values.
+		The heather is boardctl.h in arch/amr/include/stm32h745 directory.
```

I think you meant to write header, could you please check.
```suggestion
		The header is boardctl.h in arch/amr/include/stm32h745 directory.
```
Also I think the second line of description is not required as we may use it for other boards in the future.

---

@ewoodev Could you check this comment?

---

Thanks, I think so. I deleted the line.